mkosi/__init__.py

```diff
@@ -168,6 +170,28 @@ class MkosiException(Exception):
     """Leads to sys.exit"""
 
 
+def dictify(f):
+    def wrapper(*args, **kwargs):
+        return dict(f(*args, **kwargs))
+    return functools.update_wrapper(wrapper, f)
+
+@dictify
+def read_os_release():
+    try:
+        f = open('/etc/os-release')
+    except FileNotFoundError:
+        f = open('usr/lib/os-release')
```

Comment:
I don't really see that as a grave problem, but this is less robust than before, since it doesn't catch a `FileNotFoundError` if both files don't exist. Previously the defaults would have been used.

Reply:
That's somewhat on purpose: os-release is supposed to always exists (and it's going to be there on any system released in the last 8 years...)
So I though that this function should return an error. The error is still caught in detect_distribution(), so effectively you still get `None, None` as the answer.